packages/react-native-oursky/src/types.js

```diff
@@ -6,3 +6,5 @@ export type ListRenderItemInfo<itemT> = {
   item: itemT,
   separators: SeparatorsObj,
 };
+
+export type RefObject<T> = { current: null | T };
```

Comment:
Actually it isn't quite correct because in React ref can be three things.

```typescript
type Ref<T> =
    string // string ref, it is deprecated
  | (r: T | null) => void // function ref
  | { current: null | T } // ref object
```

Flow should have provided the definition of `Ref`

Reply:
https://github.com/facebook/flow/blob/v0.78.0/lib/react.js#L189